apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/StorageClient.cs

```diff
@@ -167,6 +132,29 @@ public UrlSigner UrlSigner
                 UnauthenticatedAccess = true
             }.Build();
 
+        /// <summary>
+        /// Creates a URL signer built base on this client, that is using the same credential as this client, as well
+        /// as defaulting to this client's URI scheme, host and port. If the credential used by this client is not compatible
```

Comment:
Possibly just use `<exception>` instead of summary for the exception bit? (I don't mind either way.)